### ISSUE TITLE
Get scheduled notification by id (former method "isScheduled")

### ISSUE THREAD

**Issue by mayconight** [ionic-team/capacitor-plugins#1272]

**Comment by joshuavillano** [comment]
Is that different from LocalNotifications.getPending()?

**Comment by mayconight** [comment]
@joshuavillano yes. I edited the title and description, because "isScheduled" actually returns a Promise<boolean>.

**Comment by eric-horodyski** [comment]
This thread has been inactive for some time, and we’re cleaning up older items to keep the backlog focused and manageable.

Details on the effort:
[https://ionic.io/blog/keeping-capacitors-backlog-healthy](https://ionic.io/blog/keeping-capacitors-backlog-healthy)

If this is still relevant today, feel free to open a new issue with current information and we’ll review it.
